packages/now-static-build/src/index.ts

```diff
@@ -159,25 +166,27 @@ export async function build({
       const { HUGO_VERSION, ZOLA_VERSION, GUTENBERG_VERSION } = process.env;
 
       if (HUGO_VERSION && !meta.isDev) {
-        /*
         const [major, minor] = HUGO_VERSION.split('.').map(Number);
         const isOldVersion = major === 0 && minor < 43;
         const prefix = isOldVersion ? `hugo_` : `hugo_extended_`;
-        TODO: We can't currently use extended because of `libstdc++.so.6` needs to be updated.
```

Comment:
The only functional change here is that this code was uncommented from a previous PR.

The tests already exist for both "old" and "new" hugo.

All the other changes are from lint.